packages/angular/projects/ui-angular/src/lib/primitives-poc/flex/flex-component/flex.component.html

```diff
@@ -0,0 +1,11 @@
+<div
+  class="amplify-flex"
+  [ngStyle]="{
+    'flex-flow': this.direction + ' ' + this.wrap,
+    'place-content': this.justifyContent + ' ' + this.alignContent,
+    'align-items': this.alignItems,
```

Comment:
Should we consider expanding these shorthands? That'll allow customers to override them easily if they're using css directly:

```suggestion
    'flex-direction': direction,
    'flex-wrap': wrap,
    'align-content': 'alignContent',
    'place-content': 'place-content',
```